src/content/docs/2/guide/notification.mdx

```diff
@@ -0,0 +1,193 @@
+---
+title: Notifications
+description: Send native notifications to the user.
+---
+
+import PluginBreadcrumb from '@components/PluginBreadcrumb.astro';
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+import CommandTabs from '@components/CommandTabs.astro';
+import Stub from '@components/Stub.astro';
+
+<PluginBreadcrumb
+	version="v2.0.0-alpha.2"
+	github="https://github.com/tauri-apps/plugins-workspace/tree/v2/plugins/notification"
+	npm="https://www.npmjs.com/package/@tauri-apps/plugin-notification"
+	cratesio="https://crates.io/crates/tauri-plugin-notification"
+	jsApi="/2/reference/js/notification"
+	docsrs="https://docs.rs/tauri-plugin-notification/2.0.0-alpha.2/tauri_plugin_notification/"
+/>
+
+Send native notifications to your user using the notification plugin.
+
+{/* TODO: Demo component */}
+
+### Security Considerations
+
+Aside from normal sanitization procedures of user input there are currently no known security considerations.
+
+## Setup
+
+First you have to install the notification plugin in the backend.
+
+<Tabs>
+    <TabItem label="Automatic">
+
+    Use the [add command](/2/reference/cli/#plugin-add) in the Tauri CLI to add the plugin:
+
+    ```bash
+    tauri plugin add notification
+    ```
+
+    {/* TODO: Does this also add it in `lib.rs`? Or does a step need to be added to cover that? */}
+
+    </TabItem>
+    <TabItem label="Manual">
+
+    1. Run `cargo add tauri-plugin-notification` to add the plugin to the project's dependencies in `Cargo.toml`.
+
+    2. Modify `lib.rs` to initialize the plugin:
+
+    ```rust
+    // lib.rs
+    #[cfg_attr(mobile, tauri::mobile_entry_point)]
+    pub fn run() {
+        tauri::Builder::default()
+            // Initialize the plugin
+            .plugin(tauri_plugin_notification::init())
+            .run(tauri::generate_context!())
+            .expect("error while running tauri application");
+    }
+    ```
+
+    If you'd like to use notifications in JavaScript then install the npm package as well:
+
+    <CommandTabs
```

Comment:
I fully realise this goes against my own UX principals so feel free to rake me across the coals if you don't think it works here @simonhyll 😅

Reply:
I should screenshot this and post on the Astro server 😂 

It certainly makes more sense to do it like that if you structure things without the separated Frontend and Backend sections. As I see it it's not *actually* nested tabs, it's just the ability to switch which package manager you wanna use

Speaking of choosing your package manager, what if when the user switches tabs we save that to LocalStorage so all CommandTabs default to the last one they picked?

Reply:
> I should screenshot this and post on the Astro server 😂

Please do, I'm sure Chris would get a kick out of it 😆 

> Speaking of choosing your package manager, what if when the user switches tabs we save that to LocalStorage so all CommandTabs default to the last one they picked?

I believe that's what they've discussed in https://github.com/withastro/starlight/issues/462. If you get bored and want to dive into Starlight land I'm sure HiDeoo would love to have a contribution based on his issue.